prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -122,6 +124,8 @@ postsubmits:
     annotations:
       testgrid-dashboards: googleoss-test-infra
       testgrid-tab-name: knative-prow-deploy
+      testgrid-alert-email: k8s-infra-oncall@google.com
+      testgrid-num-failures-to-alert: '1'
```

Comment:
1 will not distinguish an outage from flakiness. I'd vote for making these 2 or 3.

Reply:
Makes sense, up postsubmit to be 2 and periodic to be 3